pkgs/irc-announce/default.nix

```diff
@@ -32,7 +32,12 @@
   # echo2 and cat2 are used output to both, stdout and stderr
   # This is used to see what we send to the irc server. (debug output)
   echo2() { echo "$*"; echo "$*" >&2; }
-  cat2() { tee /dev/stderr; }
```

Comment:
The code already does this.
See `tee(1)`:

```
DESCRIPTION
       Copy standard input to each FILE, and also to standard output.
```

Reply:
/dev/stderr is not a file in a systemd service

Reply:
References:
- https://cgit.krebsco.de/stockholm/commit/krebs/5pkgs/simple/irc-announce/default.nix?id=5281ea110297a40707a4b94855d5d3da04c71e37
- https://github.com/poettering/systemd/commit/c9cc1a2d0c591e571f0e27dff1e9c03d341a0d7d